source/includes/tutorial-setup.rst

```diff
@@ -0,0 +1,151 @@
+.. _kafka-tutorials-setup:
+
+Before You Get Started
```

Comment:
I noticed this wasn't in the TOC yet -- Just wanted to highlight this in case you just forgot

Reply:
Sorry I didn't communicate very clearly, but Robert and I haven't decided whether this is a separate content item or an include in the three tutorials for which they apply.